### ISSUE TITLE
script packages providing additional image metadata

### ISSUE THREAD

**Issue by jmtd** [jboss-dockerfiles/dogen#93]
If a given script package adds functionality to the image that is tuned by environment variables, then it would be better if the definition of those variables was close to the script package, rather than being in the image yaml. Therefore, a script package shared amongst many images will have the environment variables defined in one place only. Otherwise, they must be duplicated amongst all the image.yaml files for images that share the script package. This may also apply to other metadata (ports perhaps?)

**Comment by goldmann** [comment]
Yes, @l-d-j already has answer to this is already part of `module.yaml`.
